matlabMeas/MeasurementUtilities/sweep2DMeasSR830_Func.m

```diff
@@ -66,19 +67,22 @@
         
     end
     
-    evalin("base","DACGUI.updateDACGUI");
-    drawnow;
+    if haveDAC
+        evalin("base","DACGUI.updateDACGUI");
+        drawnow;
+    end
     pause(timeBetweenPoints);
-    
-    if mod(currentMainLoopIter, 2) == 1
-        paramVector2Actual = paramVector2;
+
+    if mod(valueIndex1, 2) == 1
+        valueIndexVector2 = 1:length(paramVector2);
     else
-        if mod(currentMainLoopIter, 2) == 0
-            paramVector2Actual = flippedParam2;
+        if mod(valueIndex1, 2) == 0
```

Comment:
this if statement seems redundant. mod(a,2) can only have 2 values, 1 or 0. You already check the case of mod(a,2) == 1. If it is not true, it's obviously going to be 0?

Reply:
Good point. This will be changed to be more efficient.